src/path-transformers.ts

```diff
@@ -85,6 +96,9 @@ export function urlFromPath(root: string, target: string): string {
 
   const relativePath = path.posix.relative(root, target);
 
+  // The startsWith(root) check is important on Windows because of the case
+  // where paths have different drive letters.  The startsWith('../') will
+  // catch the general not-in-root case.
```

Comment:
Would it be clearer to catch the windows issue here by asserting that `relativePath` isn't absolute?